assemblies/assembly_openshift-ansible-playbook/main.adoc

```diff
@@ -30,6 +32,9 @@ IMPORTANT: ReadWriteMany mode is not supported on OpenShift Online and OpenShift
 include::{enterprise-dir}/openshift/ba-openshift-overview-con.adoc[leveloffset=+1]
 include::{enterprise-dir}/openshift/dm-openshift-prepare-con.adoc[leveloffset=+1]
 include::{enterprise-dir}/openshift/imagestreams-file-install-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/openshift/secrets-kie-create-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/openshift/secrets-central-create-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/openshift/externaldb-build-proc.adoc[leveloffset=+2]
```

Comment:
For External DB configuration I would rather expect link to another documentation, rather then including this chapter here. This configuration is optional. In this case I would prefere shorter document with all necessary configuration and links to additional possible configuration.
In same way could be handled glusterds chapter.

wdyt?